lab1/app/src/main/java/com/example/labyrinth/view/DrawView.java

```diff
@@ -12,28 +12,36 @@
 
 import static java.lang.Math.min;
 
-enum Type {
-    Classic, Hard
-}
+import com.example.labyrinth.Cell;
+import com.example.labyrinth.Labyrinth;
+import com.example.labyrinth.Type;
 
 @SuppressLint("ViewConstructor")
-public class DrawView extends View {
+public abstract class DrawView extends View {
```

Comment:
Refactor DrawView class. Make it abstract class, and some methods with different logic should be overriden in subclasses